Cangjie5.txt

```diff
@@ -72475,6 +72478,7 @@
 𡄩	rbvp
 𡆟	rbvu
 𡅘	rbvv
+𠶼	rbvv
```

Comment:
May be `rbuv` to match [𥄉](http://chidic.eduhk.hk/v.php?id=4007), [𠸵](http://chidic.eduhk.hk/v.php?id=47700).

Reply:
done.